packages/skilllint/version.py

```
@@ -0,0 +1,39 @@
+
+def _get_hatch_version() -> str | None:
+    try:
+        from hatchling.metadata.core import ProjectMetadata
+        from hatchling.plugin.manager import PluginManager
+        from hatchling.utils.fs import locate_file
+    except ImportError:
+        return None
+
+    pyproject_toml = locate_file(__file__, "pyproject.toml")
+    if pyproject_toml is None:
+        raise RuntimeError("pyproject.toml not found although hatchling is installed")
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)